app/assets/admin/js/settings.js

```diff
@@ -917,6 +943,20 @@ jQuery( document ).ready( function ( $ ) {
 	jQuery( "#rtm-masonry-change-thumbnail-info" ).click( function ( e ) {
 		jQuery( "html, body" ).animate( { scrollTop: 0 }, '500', 'swing' );
 	} );
+
+	jQuery( '#rtm-export-button' ).click( function () {
+		data = {
+			action: "rtmedia_export_settings",
+		};
+		jQuery.post( ajaxurl, data, function ( data ) {
+			var dataStr            = "data:text/json;charset=utf-8," + encodeURIComponent( JSON.stringify( data ) );
+			var downloadAnchorNode = document.createElement( 'a' );
+			downloadAnchorNode.setAttribute( 'href', dataStr );
+			downloadAnchorNode.setAttribute( 'download', 'rtm-settings' + '.json' );
```

Comment:
You don't need to concat two static strings.  `downloadAnchorNode.setAttribute( 'download', 'rtm-settings.json' );`